### PR TITLE
Update compile.sh

### PR DESCRIPTION
**Pull request by SphinxC0re**

### CODE REVIEW

File: compile.sh [hunk 1]
```diff
@@ -180,13 +179,13 @@ echo
...
 popd
-pushd MCServer/MCServer
+pushd cuberite/cuberite
 echo
 echo "-----------------"
 echo "Compilation done!"
 echo
 echo "Cuberite awaits you at:"
-echo "`pwd`/MCServer"
+echo "`pwd`/Cuberite"
```
Comment (bearbin) [inline compile.sh]: I think this line might be wrong.

Comment (SafwatHalaby) [inline compile.sh]: If the output executable name is still MCServer, then this is indeed wrong (until the output executable name is updated).

File: compile.sh [hunk 2]
```diff
@@ -194,5 +193,5 @@ exit 0
 
 :windows_detected
 echo "This script is not available for Windows yet, sorry."
-echo "You can still download the Windows binaries from: http://mc-server.org"
+echo "You can still download the Windows binaries from: http://cuberite.org"
```
Comment (SafwatHalaby) [inline compile.sh]: https? :D

File: compile.sh [hunk 3]
```diff
@@ -3,7 +3,7 @@
 # Do we already have a repo?
 if [[ -d .git && -f easyinstall.sh && -f MakeLuaAPI.cmd ]]; then # A good enough indicator that we're in the MCServer git repo.
```
Comment (SafwatHalaby) [inline compile.sh]: "A good enough indicator that we're in the Cuberite ~~MCServer~~ git repo."

**Comment by SafwatHalaby** [comment]
Merge?

**Comment by SphinxC0re** [comment]
No, I still want to apply the comments

**Comment by SafwatHalaby** [comment]
Note that you've mixed in some unrelated changes to other files.

**Comment by SphinxC0re** [comment]
Yeah, @bearbin added some chnagfes to the branch

**Comment by bearbin** [comment]
I haven't pushed anything in the last few weeks - could you point me to where I added some changes?

**Comment by tigerw** [comment]
### Sleep-pushing (*noun*, `technical`)
#### Pronunciation:  /sliːp/ /pʊʃ‐ɪn/
----
The act of committing and pushing changes to the popular online service GitHub, whilst one is asleep.

*See also: sleep-walking*

File: compile.sh [hunk 4]
```diff
@@ -1,11 +1,11 @@
 #|| goto :windows_detected
 
 # Do we already have a repo?
-if [[ -d .git && -f easyinstall.sh && -f src/BlockArea.cpp ]]; then # A good enough indicator that we're in the MCServer git repo.
+if [[ -d .git && -f easyinstall.sh && -f MakeLuaAPI.cmd ]]; then # A good enough indicator that we're in the Cuberite git repo.
```
Comment (madmaxoft) [inline compile.sh]: `MakeLuaAPI.cmd` has been removed and I have changed the check to `src/BlockArea.cpp`, why are you reverting my change back?

**Comment by SphinxC0re** [comment]
:grin: I'm not

**Comment by tigerw** [comment]
*laughs nervously*

**Pull request closed without merging**